src/lib/Pagination/Mapper/AbstractPagerContentToDataMapper.php

```diff
@@ -113,11 +113,13 @@ protected function setTranslatedContentTypesNames(array &$data, array $contentTy
             $this->userLanguagePreferenceProvider->getPreferredLanguages()
         );
 
+        $contentTypesArray = [...$contentTypes];
```

Comment:
Note: fix for phpstan

Reply:
Not sure what was the problem with PHPStan but `ContentTypeService::loadContentTypeList` already returns Content Type ID indexed array.

Reply:
`loadContentTypeList` returns `iterable` and the error was _Cannot access offset mixed on iterable<......>._
But I realised that doing `[...]` is not a good idea.